### PR TITLE
Update for Arabic subs

### PR DESCRIPTION
**Pull request by WaseemDabbas**

### CODE REVIEW

File: config/locales/ar.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ ar:
   groups:
     group_header: اعثر على أقرب مجموعة إليك
     learn_more: اعرف أكثر عن هذه المجموعة
-    next_event: "Next event: %{event_name} (in %{time_in_future})"
+    next_event: "الفعاليات القادمة: %{event_name} (in %{time_in_future})"
```
Comment (creature) [inline config/locales/ar.yml]: The word 'in' within brackets needs to be translated in this string too; this template gets rendered as "Next event: Intro meeting (in 3 days)". You might also need to rearrange where the brackets are in this sentence too. `time_in_future` will automatically handle rendering strings like "3 days" in Arabic. 

```
next_event: "الفعاليات القادمة: %{event_name} (in %{time_in_future})"
                                               ↑↑
```

File: config/locales/ar.yml [hunk 2]
```diff
@@ -2,19 +2,19 @@ ar:
...
+    view_in_arabic: عرض باللغة العربية
     footer:
-      email: Questions? Email us on <a href="mailto:localwelcometeam@gmail.com">localwelcometeam@gmail.com</a>
-      twitter: Follow Local Welcome on <a href="https://twitter.com/localwelcome">Twitter</a>
-      facebook: Like Local Welcome on <a href="https://www.facebook.com/LocalWelcome/">Facebook</a>
-      established: Local Welcome was established in 2015. Learn more <a href="%{link}">about the project</a>.
+      email: للاستفسارات? أرسل لنا على <a href="mailto:localwelcometeam@gmail.com">localwelcometeam@gmail.com</a>
+      تويتر: تابعونا على <a href="https://twitter.com/localwelcome">Twitter</a>
+      فيس بوك: تابعونا على <a href="https://www.facebook.com/LocalWelcome/">Facebook</a>
+      established: تم تأسيس Local Welcome في 2015. لمعرفة المزيد <a href="%{link}">about the project</a>.
```
Comment (creature) [inline config/locales/ar.yml]: The "about the project" part within the `<a>` tag needs to be translated here too: 

```
established: تم تأسيس Local Welcome في 2015. لمعرفة المزيد <a href="%{link}">about the project</a>.
                                                                             ↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑
```

**Comment by creature** [comment]
Thanks loads for these! There's a couple of incomplete translations that I commented on inline, and a couple of things that I can fix up (the key – that is, the text before the colon – should never be translated, but `twitter:` and `facebook:` was). If you can fix the incomplete ones, I'll get this merged and deployed. :)

File: config/locales/ar.yml (hunk 1 above)
Comment (WaseemDabbas) [inline config/locales/ar.yml]: next_event: "الفعاليات القادمة: %{event_name} (في %{time_in_future})"

File: config/locales/ar.yml (hunk 2 above)
Comment (WaseemDabbas) [inline config/locales/ar.yml]: established:في 2015 Local welcome تم تأسيس
لمعرفة المزيد <a href="%{link}">حول المشروع</a>

Comment (WaseemDabbas) [inline config/locales/ar.yml]: <a href="mailto:localwelcometeam@gmail.com">localwelcometeam@gmail.com</a> للاستفسارات؟ أرسل لنا على 
 <a href="https://twitter.com/localwelcome">Twitter</a> تابعونا على
 <a href="https://www.facebook.com/LocalWelcome/">Facebook</a> تابعونا على
في 2015 Local Welcome تم تأسيس
لمعرفة المزيد <a href="%{link}">حول المشروع</a>

**Comment by WaseemDabbas** [comment]
I did complete what you've asked.
If there's any update required, please let me know :D

**Comment by WaseemDabbas** [comment]
I did complete what you've asked.
If there's any updates required, please let me know :D

**Comment by creature** [comment]
I've merged this pull request now. Thanks for the help! :)

**Pull request merged**